src/js/components/Markdown/README.md

```diff
@@ -17,6 +20,10 @@ Custom components and props to override html elements such as 'img'
       or 'pre'. By default 'a', 'p', 'img', and table elements are overridden
       with grommet components
 
+**options**
+
+You can see all the options [here](https://github.com/probablyup/markdown-to-jsx).
+
```

Comment:
The README file is auto-generated when running `yarn test-update` and it shouldn't be manually edited.
A way to achieve it is to add `options` prop on the doc.js file and then auto-generate the readme to include the new prop details.